### DISCUSSION TITLE
Gradient

### DISCUSSION THREAD

**Discussion by pamelahu** [twilio-labs/paste#3262] [General] [answered]
![Screenshot 2023-06-05 at 2 42 28 PM](https://github.com/twilio-labs/paste/assets/18605252/2d47d34b-fd26-404d-9cba-8034368c2f08)

Hi, Liberty helped design this beautiful branding for One Admin. My engineer isn't sure how to get that gradient effect under the mast head - any tips?

**Comment by loreina** [comment]
Hi @pamelahu,

I spoke with @kendal-kirby earlier this quarter and with priority shifts, we decided to leave the One Admin border out of scope for Navigation. Has there been a change in priorities and will the team now need the border? Thanks!

**Comment by pamelahu** [comment]
We want to ship a first iteration of One Admin which would've included the
gradient border. BUT we don't *need* to have it this time. Do you know
where the border falls on the roadmap?

On Tue, Jun 6, 2023 at 12:35 PM Loreina Chew ***@***.***>
wrote:

> Hi @pamelahu
> <https://urldefense.com/v3/__https://github.com/pamelahu__;!!NCc8flgU!ZX3GglpcQVOROgahXAUjshycklCpPQjOa2vp2VBEdaw1Px5lY0jWpb2aD3-0wJKfbg3qfBZLAamluFCYcMiO$>
> ,
>
> I spoke with @kendal-kirby
> <https://urldefense.com/v3/__https://github.com/kendal-kirby__;!!NCc8flgU!ZX3GglpcQVOROgahXAUjshycklCpPQjOa2vp2VBEdaw1Px5lY0jWpb2aD3-0wJKfbg3qfBZLAamluC_NFlRm$>
> earlier this quarter and with priority shifts, we decided to leave the One
> Admin border out of scope for Navigation. Has there been a change in
> priorities and will the team now need the border? Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/twilio-labs/paste/discussions/3262*discussioncomment-6102868__;Iw!!NCc8flgU!ZX3GglpcQVOROgahXAUjshycklCpPQjOa2vp2VBEdaw1Px5lY0jWpb2aD3-0wJKfbg3qfBZLAamluIERkIwe$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AEN6JRGTM7LG6V7EORKTNUTXJ5L3TANCNFSM6AAAAAAY3LNR44__;!!NCc8flgU!ZX3GglpcQVOROgahXAUjshycklCpPQjOa2vp2VBEdaw1Px5lY0jWpb2aD3-0wJKfbg3qfBZLAamluKmpIHqe$>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Pamela Hu
Product Designer, Twilio.org
[image: Twilio] <https://www.twilio.com/?utm_source=email_signature>

**Comment by loreina** [comment]
Hey @pamelahu,

@kendalkirby let us know that there's a One Admin release in August that you'd all like the gradient border and light sidebar for ideally. Our team has release planning next week so we'll discuss of our upcoming priorities, and let you know if we're able to meet that target date.

Could you fill out both the [Comms BU Dependency Sheet](https://docs.google.com/spreadsheets/d/1O3UFLbmT7cYUlrsTQ3jp8eJUXUhnEHjdw7k-bMFmpOw/edit#gid=1962409093) and the [Paste Dependency Tracker](https://airtable.com/shrvMvL3BG6Zm0f9l)? These are the official means for incoming requests and helps our team stay accountable for our work. Thanks!

**Comment by pamelahu** [comment]
@loreina Will do! Also, in general, is there an ETA for the navigation UI kit component? My engineer is asking about it but I see that it's still marked as "in progress" in the Paste roadmap.

**Reply by loreina** [reply]
Navigation components will be ready for our [June 27 release](https://paste.twilio.design/roadmap#2023-06-27).

**Comment by SiTaggart** [comment, marked as answer]
Hi there

Whilst we might not get to it in time for launch, you can create the gradient using some css and referencing our design tokens yourself. It's basically what we would do anyway, just that it will love in the Application until we ship the fully owned and supported version.

https://codesandbox.io/s/gradient-bar-example-fzs2zc?file=/src/index.tsx

Let us know how you go!
